src/main/java/mate/academy/Main.java

```diff
@@ -27,29 +50,19 @@ public static void main(String[] args) {
         secondCinemaHall.setCapacity(200);
         secondCinemaHall.setDescription("second hall with capacity 200");
 
-        CinemaHallService cinemaHallService = null;
         cinemaHallService.add(firstCinemaHall);
         cinemaHallService.add(secondCinemaHall);
 
-        System.out.println(cinemaHallService.getAll());
-        System.out.println(cinemaHallService.get(firstCinemaHall.getId()));
-
         MovieSession tomorrowMovieSession = new MovieSession();
-        tomorrowMovieSession.setCinemaHall(firstCinemaHall);
-        tomorrowMovieSession.setMovie(fastAndFurious);
+        tomorrowMovieSession.setCinemaHall(firstCinemaHall); // firstCinemaHall створений раніше
+        tomorrowMovieSession.setMovie(fastAndFurious); // fastAndFurious створений раніше
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
-
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
-
-        MovieSessionService movieSessionService = null;
         movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
 
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
-        System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+        shoppingCartService.addSession(tomorrowMovieSession, user);
+
+        ShoppingCart shoppingCart = shoppingCartService.getByUser(user);
```

Comment:
Ensure that the `getByUser` method in `ShoppingCartService` is necessary and correctly implemented. It was previously noted that a similar method in `OrderDao` was redundant. Verify that this method is not overlapping with any other existing method and is required for the functionality.